### ISSUE TITLE
npm test fails on Windows

### ISSUE THREAD

**Issue by swar30** [coatue-oss/angular2react#27]
When running `npm test` on windows failing with

```
$ npm test

> angular2react@2.2.4 pretest C:\git\angular2react
> npm run build


> angular2react@2.2.4 build C:\git\angular2react
> npm run clean; tsc -d && mv ./index.js ./index.es2015.js && tsc -t es5

npm info it worked if it ends with ok
npm info using npm@5.5.1
npm info using node@v8.9.3
npm ERR! missing script: clean;

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\shnayder\AppData\Roaming\npm-cache\_logs\2018-02-19T19_16_18_515Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! angular2react@2.2.4 build: `npm run clean; tsc -d && mv ./index.js ./index.es2015.js && tsc -t es5`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angular2react@2.2.4 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\shnayder\AppData\Roaming\npm-cache\_logs\2018-02-19T19_16_18_535Z-debug.log
npm ERR! Test failed.  See above for more details.
```

This seems to fix it

```
diff --git a/package.json b/package.json                                                  
index ae34dcc..7a98b28 100644                                                             
--- a/package.json                                                                        
+++ b/package.json                                                                        
@@ -6,7 +6,7 @@                                                                           
   "main:esnext": "index.es2015.js",                                                      
   "typings": "index.d.ts",                                                               
   "scripts": {                                                                           
-    "build": "npm run clean; tsc -d && mv ./index.js ./index.es2015.js && tsc -t es5",   
+    "build": "npm run clean && tsc -d && mv ./index.js ./index.es2015.js && tsc -t es5", 
     "clean": "rm -f ./*.d.ts; rm -f ./*.map",                                            
     "pretest": "npm run build",                                                          
     "prepublishOnly": "npm test",                                                        
```

does this make sense on Linux ?

**Comment by bcherny** [comment]
Fixed in 3.0.0.

**Comment by swar30** [comment]
appreciated, thanks
